### PR TITLE
Added Spanish translations to sample project

### PR DESCRIPTION
**Pull request by voghDev**
In relation with #292, I've also translated the strings from the sample project. Please have a look at the PR and let me know your opinion.

Thanks!

### CODE REVIEW

**Comment by Andrew-Quebe** [comment]
I'm not merging this until I finish the example app...just fyi.

**Comment by voghDev** [comment]
Ok, let me know when more translations are needed then I send you a final PR

**Comment by paolorotolo** [comment]
@Andrew-Quebe any update on this?

**Comment by voghDev** [comment]
May I review the translations again and send a new PR?

**Comment by avluis** [comment]
@voghDev You definitely can -- you can either rebase with the master branch or close this PR and start fresh.

File: example/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <string name="default_intro2">Este es el segundo ejemplo de la librería AppIntro. Puede llevarse a cabo con muy pocas líneas de código, y este es el segundo tipo de layout que ofrece la librería. Pulsa en el botón de abajo para iniciar la demo.</string>
+    <string name="custom_layout_intro">Aquí se demuestra como se puede usar un layout personalizado en la librería para tener un mayor nivel de personalización. Pulsa en el botón de abajo para iniciar la demo.</string>
+    <string name="custom_background_intro">This demonstrates the ability to set custom backgrounds for different slides. Useful if you want images vs. colors in your backgrounds. Tap the button below to launch the demo.</string>
+    <string name="custom_background_intro">Aquí se demuestra como se puede poner un fondo personalizado para cada slide. Esto es util si quieres usar colores contrastando con imagenes para tus fondos. Pulsa en el botón de abajo para iniciar la demo.</string>
```
Comment (fanky10) [inline example/src/main/res/values-es/strings.xml]: this is duplicated, I think it can bring a compilation error, please leave the translated one.

**Comment by fanky10** [comment]
I just left a comment, in regards duplicate entry. Good job with the tildes, you did not miss a single one! (:

File: example/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (voghDev) [inline example/src/main/res/values-es/strings.xml]: True! gonna fix it :-)

**Comment by fanky10** [comment]
LG2M

**Pull request merged**